scripts/update-ndc-postgres.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
I suggest failing fast.

```suggestion

set -e
set -u
set -o pipefail

```